LibGAP.jl/src/julia_to_gap.jl

```diff
@@ -0,0 +1,99 @@
+## Converters
+
+## Default
+julia_to_gap(x::GAPInputType, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = x
+
+
+## Integers
+julia_to_gap(x::Int128, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = MakeObjInt(BigInt(x)) # FIXME: inefficient hack
+#julia_to_gap(x::Int64)  = x
+julia_to_gap(x::Int32, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = Int64(x)
+julia_to_gap(x::Int16, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = Int64(x)
+julia_to_gap(x::Int8, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = Int64(x)
+
+## Unsigned Integers
+julia_to_gap(x::UInt128, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = MakeObjInt(BigInt(x)) # FIXME: inefficient hack
+julia_to_gap(x::UInt64, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = MakeObjInt(BigInt(x)) # FIXME: inefficient hack
+julia_to_gap(x::UInt32, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = Int64(x)
+julia_to_gap(x::UInt16, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = Int64(x)
+julia_to_gap(x::UInt8, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive  = Int64(x)
+
+## BigInts
+julia_to_gap(x::BigInt, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive = MakeObjInt(x)
+
+## Rationals
+function julia_to_gap(x::Rational{T}, ::Val{Recursive}=Val(false), recursion_dict = nothing) where Recursive where T <: Integer
+    denom_julia = denominator(x)
+    numer_julia = numerator(x)
```

Comment:
According to Codecov, there are no tests for this function at all, it is never called :-(